### PR TITLE
Applying visual redesign styles to fields

### PR DESCRIPTION
**Pull request by dlockhart**
@dbatiste, @njostonehouse: interested in your feedback. Here's a screenshot of  what the test page looks like with the new styles applied:
a screen shot

### CODE REVIEW

**Comment by njostonehouse** [comment]
Do we deal with multi-line (wrapping) inline field labels?

**Comment by dlockhart** [comment]
There wasn't anything before nor is there anything now to prevent the text in field labels from wrapping.

File: field.scss [hunk 1]
```diff
@@ -18,3 +18,30 @@
 	}
 
 }
+
+@mixin vui-field-label($inline: false) {
+
+	color: $vui-color-ferrite;
+	cursor: default;
+	display: block;
+	font-size: 0.7rem;
```
Comment (dbatiste) [inline field.scss]: Perhaps make use of vui-typography mixin [vui-typography-small-text](https://github.com/Brightspace/valence-ui-typography/blob/master/typography.scss#L97) - I suspect the intent was to use that style for labels, but I could be wrong.  Besides eliminating the need to specify color and size, it also specifies character spacing which is not included here.  The mixin allows overriding the margin, but you'd have to add line-height as a param if you were to use it.

**Comment by dlockhart** [comment]
@dbatiste: thanks for the suggestion -- I've updated it to use `vui-typography`'s "small text" mixin.

**Pull request merged**